x-pack/test/functional/apps/upgrade_assistant/upgrade_assistant.ts

```diff
@@ -17,10 +17,11 @@ export default function upgradeAssistantFunctionalTests({
   const security = getService('security');
   const log = getService('log');
   const retry = getService('retry');
+  const testSubjects = getService('testSubjects');
 
-  // Failing: See https://github.com/elastic/kibana/issues/86546
-  describe.skip('Upgrade Checkup', function () {
-    this.tags('includeFirefox');
+  // Updated for the hiding of the UA UI.
```

Comment:
[nit] I might modify this comment to provide more context as to why some of the tests are skipped. Something like:

```
// UA is currently disabled by default. Some tests are skipped, but should be re-enabled for the last minor of the next major release.
```